### PR TITLE
Task: Get Avatar functionality to child composites in meeting composite

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What
add props to child components for avatar customization.

# Why
Allows custom avatar overrides in existing composite functionality in meeting composite.

https://skype.visualstudio.com/SPOOL/_workitems/edit/2694402
# How Tested
Ran meeting sample with custom avatar information in the meeting sample.

### CODE REVIEW

File: samples/Meeting/src/app/views/MeetingScreen.tsx [hunk 1]
```diff
@@ -77,6 +77,10 @@ export const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
   if (!adapter) {
     return <Spinner label={'Creating adapter'} ariaLive="assertive" labelPosition="top" />;
   }
+  const onFetchAvatarPersonaData = async (userId): Promise<AvatarPersonaData> => ({
```
Comment (JamesBurnside) [inline samples/Meeting/src/app/views/MeetingScreen.tsx:80]: remove test code,
but would be good to have a UI test added for this feature

**Review by alkwa-msft — APPROVED** [review]

File: change/@internal-react-composites-2ec9225a-d4e0-42e1-a975-3a987736beb3.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "implemented custom datamodel functionality to meeting composite excluding sidebar.",
```
Comment (prprabhu-ms) [inline change/@internal-react-composites-2ec9225a-d4e0-42e1-a975-3a987736beb3.json:3]: Nitty nit: Capitalize line (Implemented...), and no trailing period.
Also, we're referring to sidebase as "side pane".

Comment (prprabhu-ms) [reply change/@internal-react-composites-2ec9225a-d4e0-42e1-a975-3a987736beb3.json:3]: From the PR, it looks like this is only implementing it for the side pane (not excluding it)?

**Review by prprabhu-ms — APPROVED** [review]

File: change/@internal-react-composites-2ec9225a-d4e0-42e1-a975-3a987736beb3.json (hunk 2 above)
Comment (dmceachernmsft) [reply change/@internal-react-composites-2ec9225a-d4e0-42e1-a975-3a987736beb3.json:3]: this is extending the existing functionality of the calling and chat composite to accept the prop from the meeting composite level. the People pane part of the side pane still needs done.

File: samples/Meeting/src/app/views/MeetingScreen.tsx (hunk 1 above)
Comment (dmceachernmsft) [reply samples/Meeting/src/app/views/MeetingScreen.tsx]: @alkwa-msft think we should make a user story for some tests for the custom data model stuff?

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**